packages/wp-env/README.md

```diff
@@ -8,7 +8,7 @@ Ensure that Docker is running, then:
 
 ```sh
 $ cd /path/to/a/wordpress/plugin
-$ npm -g i @wordpress/env
+$ npm -g i wp-env
 $ wp-env start
```

Comment:
This could be switched back to `npx wp-env start` if/when the package is published. Is it worth doing that now or waiting?